src/main/java/frc/robot/IDs.java

```diff
@@ -13,8 +14,16 @@ public class IDs {
 		PowerDistribution.ModuleType.kRev
 	);
 
+	public static class CANCodersIDs {
+
+		public static final CoreCANcoder ARM_CAN_CODER = new CoreCANcoder(-1);
```

Comment:
```suggestion
		public static final Phoenix6DeviceID ARM_CAN_CODER = new CoreCANcoder(-1);
```

there is a big difference understand it (or ask me)